game-headed/build.gradle

```diff
@@ -2,43 +2,31 @@ plugins {
     id 'application'
     id 'com.github.johnrengelman.shadow' version '4.0.4'
     id 'com.install4j.gradle' version '7.0.9'
-    id 'de.undercouch.download' version '3.4.3'
+    id 'org.openjfx.javafxplugin' version '0.0.7' apply false
 }
 
 archivesBaseName = "$group-$name"
 description = 'TripleA Headed Game Client'
 mainClassName = 'org.triplea.game.client.HeadedGameRunner'
 
 ext {
-    javaFxRuntimeUrl = 'https://github.com/triplea-game/assets/raw/master/javafx/jfxrt-1.8.0_181.jar'
     releasesDir = file("$buildDir/releases")
-    testFxVersion = '4.0.15-alpha'
 }
-
-dependencies {
-    if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
-        compileOnly remoteLib(javaFxRuntimeUrl)
+if (JavaVersion.current() == JavaVersion.VERSION_11) {
```

Comment:
Might it make sense to make this `>=` instead of `==`?

Reply:
Not sure, thought about that as well. I didn't do it this way because I'm not entirely sure how the javafx plugin works.